site/src/components/VersionsTable/VersionsTable.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { ComponentMeta, Story } from "@storybook/react"
+import { MockTemplateVersion } from "../../testHelpers/entities"
+import { VersionsTable, VersionsTableProps } from "./VersionsTable"
+
+export default {
+  title: "components/VersionsTable",
+  component: VersionsTable,
+} as ComponentMeta<typeof VersionsTable>
+
+const Template: Story<VersionsTableProps> = (args) => <VersionsTable {...args} />
+
+export const Example = Template.bind({})
+Example.args = {
+  versions: [
+    MockTemplateVersion,
+    {
+      ...MockTemplateVersion,
+      name: "test-template-version-2",
+      created_at: "2022-05-18T18:39:01.382927298Z",
+    },
```

Comment:
What's this secondary object here - another template version? Should it have a different ID?